scripts/run-tests-with-postgres.ps1

```
@@ -0,0 +1,95 @@
+[CmdletBinding()]
+param(
+    [string]$ContainerName = "gachi-test-postgres",
+    [int]$HostPort = 55432,
+    [string]$DbName = "gachi_test",
+    [string]$DbUser = "gachi_test",
+    [string]$DbPassword = "gachi_test"
```

PSScriptAnalyzer (1.25.0)

[warning] Parameter '$DbPassword' should not use String type but either SecureString or PSCredential, otherwise it increases the chance to expose this sensitive information.

Suggested fix: Set $DbPassword type to SecureString; Set $DbPassword type to PSCredential

(PSAvoidUsingPlainTextForPassword)